docassemble/ALDashboard/aldashboard.py

```diff
@@ -241,6 +246,35 @@ def init(self, *pargs, **kwargs):
         self.template_name = kwargs.get("template_name", "unknown_error")
 
 
+def install_fonts(the_font_files: DAFileList):
+    """
+    Install fonts to the server and restart both supervisor and unoconv.
+    """
+    # create the /var/www/.fonts directory if it doesn't exist
+    if not os.path.exists("/var/www/.fonts"):
+        os.makedirs("/var/www/.fonts")
+    
+    # save the DAFile to /var/www/.fonts
+    for f in the_font_files:
+        shutil.copyfile(f.path(), "/var/www/.fonts/" + f.filename)
+
+    output = ""
+    output += subprocess.run(["fc-cache", "-f", "-v"], capture_output=True, text=True).stdout
+    output += subprocess.run(["supervisorctl", "restart", "uwsgi"], capture_output=True, text=True).stdout
+    output += subprocess.run(["supervisorctl", "start", "reset"], capture_output=True, text=True).stdout
+    if get_config("enable unoconv"):
+        output += subprocess.run(["supervisorctl", "-s", "http://localhost:9001", "restart", "unoconv"], capture_output=True, text=True).stdout
+
+    return output
+
+def list_installed_fonts():
+    """
+    List the fonts installed on the server.
+    """
+    output = subprocess.run(["fc-list"], capture_output=True, text=True).stdout
```

Comment:
The default listing isn't in any sort of order (which is weird IMO). I'd suggest piping the output through `sort`. 
```suggestion
    fc_list = subprocess.run(["fc-list"], stdout=subprocess.PIPE)
    output = subprocess.run(["sort"], stdin=fc_list.stdout, capture_output=True, text=True).stdout
    fc_list.stdout.close()
```

From https://docs.python.org/3/library/subprocess.html#replacing-shell-pipeline